scripts/fgi

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
```

Comment:
I'd even go so far as to say this should just be written in Rust :D

Reply:
This should definitely be in a script IMO, so we can run it easily and not require re-compiling, esp since this is not meant to be run in an image currently. The functionality of the script is fairly simple, it's just that long shell scripts are harder to maintain :p

I'll look into a way to write this in a portable way in python..